testfm/constants.py

```diff
@@ -9,6 +9,8 @@
 DOGFOOD_URL = settings.subscription.dogfood_url
 HOTFIX_URL = settings.testfm.hotfix_url
 SERVER_HOSTNAME = settings.testfm.server_hostname
+REPOS_HOSTING_URL = settings.testfm.repos_hosting_url
+FAKE_YUM0_REPO = f"{REPOS_HOSTING_URL}/fake_yum0/"
```

Comment:
single quotest prefrred here.

Reply:
The rest of the file is double-quoted, so I wanted to keep the current style. Shall I change it everywhere in constants.py?

Reply:
Your call, you can skip if you want :)